### ISSUE TITLE
Proposal: Rename InstallConfig.Machines to InstallConfig.MachinePools

### ISSUE THREAD

**Issue by dgoodwin** [openshift/installer#478]
In the interest of locking down this spec as much as possible, it feels like we should not have a field named "Machines" contain something that is a MachinePool, when Machine is actually a type that exists in our system.

**Comment by wking** [comment]
I preferred your suggestion [here][1] ;).  Can we close this issue in favor of that one?

[1]: https://github.com/openshift/installer/issues/264#issuecomment-422349518

**Comment by dgoodwin** [comment]
Yes I'd definitely be good with that. If so we will follow suit on our side in Hive.
